Sources/SQLKit/SQLRow.swift

```diff
@@ -7,10 +7,19 @@ public protocol SQLRow {
 }
 
 extension SQLRow {
-    public func decode<D>(model type: D.Type, prefix: String? = nil) throws -> D
+    public func decode<D>(model type: D.Type, keyPrefix: String? = nil, keyDecodingStrategy: SQLRowDecoder.KeyDecodingStrategy = .useDefaultKeys) throws -> D
```

Comment:
Out of pure curiosity, why rename `prefix` to `keyPrefix`?

Reply:
I thought this was more consistent for naming, but I can change it back.

Reply:
Since we're in RC phase now we should keep this as `prefix`.